cdap-ui/app/cdap/components/AbstractWidget/index.js

```diff
@@ -42,8 +42,9 @@ export default class AbstractWidget extends Component {
 
   render() {
     let Comp = AbstractWidgetFactory[this.props.type];
+    let {size} = this.props.widgetProps || 'large';
```

Comment:
I think this needs to be `let size = this.props.widgetProps.size || 'large';`. Currently if you don't specify a `size` then the class name is `abstract-widget-wrapper undefined`